dashboard/src/pages/components/header.ts

```diff
@@ -0,0 +1,89 @@
+import "@material/web/iconbutton/icon-button";
+import { LitElement, css, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import "@material/web/list/list";
+import "@material/web/list/list-item";
+import "@material/web/divider/divider";
+import "@material/web/button/outlined-button";
+import "../../components/ha-svg-icon";
+import { mdiArrowLeft, mdiLogout } from "@mdi/js";
+
+
+interface HeaderAction {
+  label: string;
+  icon: string;
+  action: void;
+}
+
+@customElement("dashboard-header")
+export class DashboardHeader extends LitElement {
+
+  @property() public backButton?: string;
+  @property() public actions?: HeaderAction[];
+
+  protected render() {
+    const isProductionServer = location.href.includes(":5580") || location.href.includes("hassio_ingress");
```

Comment:
This seems like a dirty solution? Also this should not be checked in every render, but only 1 time.

Reply:
yeah, I have no other way to check it for now. I'll make it an attribute of the client.